__DEFINES/spell_defines.dm

```diff
@@ -70,3 +71,9 @@
 #define SPELL_AIR 4 //Air based spells
 #define SPELL_GROUND 8 //Earthen based spells
 #define SPELL_NECROTIC 16 //Necromantic spells
+
+//Spell specializations, used for spellbook lists
+#define DEFAULT 1
```

Comment:
these should be called something like `SPELL_SPECIALIZATION_DEFAULT` because macros are global and generic names suck